arcsin.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "arcsin",
     "summary": "Inverse sine",
-    "description": "Computes the arc sine of `x`. The arc sine is the inverse function of the sine so that *`arcsin(sin(x)) = x`*.\n\nWorks on radians only.\nThe no-data value `null` is passed through and therefore gets propagated.",
```

Comment:
add return min (-pi/2) and max (pi/2) like with arccos?

Reply:
I added the range to the text for all functions. I hope that makes the ranges clearer without fiddling to much with specific decimal numbers.